### PR TITLE
Add file picker for package.json

### PR DESCRIPTION
**Pull request by fregante**
Replaces and closes #17. Reimplemented with the minimum amount of changes

Screen Shot 6

### CODE REVIEW

File: js/InfoPane.js [hunk 1]
```diff
@@ -13,8 +13,16 @@ export default function InfoPane() {
 
   const [recents, setRecents] = useState(getFileEntries());
 
+  // Handle file selection via input
+  const onSelect = (ev) => {
+    readFile(ev.target.files.item(0));
```
Comment (fregante) [inline js/InfoPane.js:18]: `input` already filters `.json` files, so it doesn't need further checks

File: js/InfoPane.js [hunk 2]
```diff
@@ -66,16 +78,17 @@ export default function InfoPane() {
 
   return html`
     <${Pane} style=${{ display: 'flex', flexDirection: 'column' }}>
+      <input id="package-input" type="file" hidden onChange=${onSelect} accept=".json"/>
       <p>
       Enter NPM module name here <i class="material-icons">arrow_upward</i> to see the dependency graph.  Separate multiple module names with commas (e.g. <a href="?q=mocha, chalk, rimraf">"mocha, chalk, rimraf"</a>).
       </p>
-      <div id="drop_target" style="text-align: center"
+      <label for="package-input" id="drop_target" style="text-align: center"
```
Comment (fregante) [inline js/InfoPane.js:85]: The whole drop target now triggers the file selection dialog when clicked

File: js/InfoPane.js [hunk 3]
```diff
@@ -66,16 +78,17 @@ export default function InfoPane() {
...
         onDrop=${onDrop}
         onDragOver=${onDragOver}
         onDragLeave=${onDragLeave}
       >
-        ... or drop a <code>package.json</code> file here
-        </div>
+        Alternatively, <button type="button">select</button> or drop a <code>package.json</code> file here
```
Comment (fregante) [inline js/InfoPane.js:90]: The button is a no-op, it's only here as a visual hint.

**Review by broofa — APPROVED** [review]
I'll test this once it's on `main` and I merge these and the other recent PRs with the port I'm doing to use `parcel`.

**Pull request merged**